schemasheets/list_element_slots.py

```diff
@@ -0,0 +1,87 @@
+import csv
+import pprint
+from typing import Dict, List
+
+import yaml
+from linkml_runtime import SchemaView
+from linkml_runtime.dumpers import yaml_dumper
+
+meta_source = "https://raw.githubusercontent.com/linkml/linkml-model/main/linkml_model/model/schema/meta.yaml"
+
+project_source = "https://raw.githubusercontent.com/microbiomedata/nmdc-schema/main/src/schema/nmdc.yaml"
+
+meta_element = "slot_definition"
+
+meta_view = SchemaView(meta_source)
+
+project_view = SchemaView(project_source)
+
+# ---
+
+slots = meta_view.class_induced_slots(meta_element)
+
+slot_names = [str(slot.alias) for slot in slots]
+
+slot_names.sort()
+
+as_if_tsv = "\t".join(slot_names)
+
+print(as_if_tsv)
```

Comment:
if this is intended as a module that code can import, there should be no side effects

If it's not a module, then remove it.

If it is useful for testing or demo purposes, make it a test or a notebook

Reply:
My experience with the Makefile was that it's required which likely speaks towards having it as a command.